lib/resources/r4/communication.yaml

```diff
@@ -0,0 +1,184 @@
+---
+name: Communication
+field_name_base_url: http://hl7.org/fhir/r4/communication-definitions.html#Communication
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  url: https://hl7.org/fhir/r4/Communication-definitions.html#Communication
+  note: resourceType must be Communication.
+  example: |
+    {
+      "resourceType": "Communication"
+    }
+  action:
+    - create
+
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: The status of the Communication.
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.status
+  action: create
+  note: As Millenniums Message Center Communicaitons are instantaneously delivered, all communications must be in a "completed" status
+  example: |
+    {
+      "status": "completed"
+    }
+
+- name: category
+  required: 'Yes'
+  cardinality: 0..1
+  type: CodeableConcept
+  description: Codeable definition of the Communication type.
+  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.category
+  note: The catgeories 'notification' and 'reminder' are supported.
+  example: | 
+    {
+      category:
+        {
+          coding: [{
+            "system": "http://terminology.hl7.org/CodeSystem/communication-category",
+            "code": "notification"
+          }]
+        }
+    }
+  action: create
```

Comment:
I think you can put terminology here too. Provenance should do both

Reply:
https://github.com/cerner/fhir.cerner.com/pull/433/commits/924ad1575295096a798f32771e85ca39f44272c6#diff-f08ae1e4c063b9a36a22adaa43fecf21R49